.gitignore

```diff
@@ -97,3 +97,6 @@ tmp.yaml
 
 .DS_Store
 Thumbs.db
+
+# secrets
+secrets.py
```

Comment:
Please add a comment to this PR about this file: what's it's content, when to add it. Also a template would be useful not to forget about all fields one should put in it.

Reply:
I guess template is not necessary - we will only paste couple of secrets to secrets.py and then deploy it on server. Hints for devops/techops would be within fabfile.py.